spec/basic.spec.js

```diff
@@ -1198,38 +1198,42 @@ describe('HtmlWebpackPlugin', () => {
     null, done, false, false);
   });
 
-  it('allows events to remove an attribute by setting it to false', done => {
```

Comment:
please keep the old test and just add 2 more - 1 for null 1 for undefined..

that way we get better error messages if 1 test fails and to guarantee that the old tests are not modified and still work